code/controllers/stonedmc/subsystem/dcs.dm

```diff
@@ -0,0 +1,6 @@
+SUBSYSTEM_DEF(dcs)
```

Comment:
I disagree with cluttering the MC tab even more with no benefit. Either make the PR actually use this, or don't port it.